.travis/tests.sh

```diff
@@ -16,8 +16,15 @@ then
         sudo python -m py_compile /home/cchq/www/travis/current/localsettings.py
     }
 
+    test_dimagi_environments() {
+        git clone https://github.com/dimagi/commcare-environments.git
```

Comment:
maybe just clone with depth = 1

Reply:
Not necessary to address this now but just noting that if we make a breaking change in this repo that requires a change to the env files then we'd need a way to change this to pull from a branch in order to make the build pass (or something like that).

Reply:
Yeah I agree with that, and I think that also highlights a sort of fundamental problem that versioning is here to solve—making this painful for us will also force us (to some extent at least) to version in a way that's mindful of others. For example, if we find ourselves needing to point this to a branch in a PR and then merge the branch in environments and then reset this to master, wait for build and merge, that's a sign that we should be creating a backwards compatibility or deprecation window so that these two repo's changes can be loosely coordinated rather than tightly coordinated. I imagine it'll also force other good patterns to the surface that I haven't yet imagined.